contracts/nova/Nova.sol

```diff
@@ -55,6 +56,12 @@ contract Nova is
         super._setPluginRegistry(_pluginRegistry);
     }
 
+    function setTrifolds(address trifolds_) external {
```

Comment:
This sets the trifold address, but I don't see any externally reachable use or logic in these changes.